docs/src/how-to/docker-nvidia-device.md

```diff
@@ -0,0 +1,32 @@
+# How to use NVIDIA GPU in Docker
+
+WSL supports NVIDIA-based GPU acceleration, making it ideal for many use cases
+including machine learning or general compute acceleration. Since Docker version
+25, the use of `virtualisation.docker.enableNvidia` has been deprecated in
+favour of a more standard specification called Container Device Interface.
+
+1. Enable the NVIDIA Container Toolkit and disable the mounting of executables,
+   as this will cause an problem related to missing libraries.
+
+```nix
+hardware.nvidia-container-toolkit = {
+  enable = true;
+  mount-nvidia-executables = false;
+};
+```
+
+2. As of Docker 25, the Docker daemon doesn't have the CDI feature enabled by
+   default.
```

Comment:
That is the default docker in nixos 24.05 already, so we can just assume everyone has 25 or older and collapse this.